src/js/eva-manager.js

```diff
@@ -24,8 +24,8 @@ if (document.location.host.indexOf('ebi.ac.uk') > -1) {
     METADATA_HOST = window.location.protocol + "//" + document.location.host + "/eva/webservices/rest";
     CELLBASE_HOST = window.location.protocol + "//" + document.location.host + "/cellbase/webservices/rest";
 } else {
-    METADATA_HOST = window.location.protocol + "//www.ebi.ac.uk/eva/webservices/rest";
-    CELLBASE_HOST = window.location.protocol + "//www.ebi.ac.uk/cellbase/webservices/rest";
+    METADATA_HOST = window.location.protocol + "//wwwdev.ebi.ac.uk/eva/webservices/rest";
+    CELLBASE_HOST = window.location.protocol + "//wwwdev.ebi.ac.uk/cellbase/webservices/rest";
```

Comment:
I see the rest of changes have been actioned, is there anything to comment about this one?